.github/workflows/render-and-document.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
   render-and-document:
     name: render-and-document
     runs-on: ubuntu-latest
+    timeout-minutes: 30        
```

Comment:
I really like this addition especially since there is a limit CRAN has to how long  the checks can run.

Reply:
Yeah, the CRAN limit is much shorter to what i remember but this is a good warning trigger if tests are starting to become really slow...